package com.zzezze.friendy.exceptions;

public class InvitationAlreadyExist extends RuntimeException {
    public InvitationAlreadyExist() {
        super("InvitationAlreadyExist");
    }
}
